### PR TITLE
Make sure trailing comma is removed

### PR DESCRIPTION
**Pull request by FinalAngel**
Before:
image

After:
image

### CODE REVIEW

File: djangocms_googlemap/models.py [hunk 1]
```diff
@@ -245,6 +245,8 @@ def get_short_description(self):
             display += '{0}, '.format(self.address)
         if self.lat and self.lng:
             display += '{0} / {1} '.format(self.lat, self.lng)
+        # remove trailing comma if lat or lng are not provided
+        display = display.rstrip(', ')
```
Comment (czpython) [inline djangocms_googlemap/models.py]: this would strip the comma regardless of lat and lng, also why add the comma in the first place if no lat/lon?

Comment (FinalAngel) [inline djangocms_googlemap/models.py]: @czpython the trailing slash is added in line 245, check screenshots in the description for how it looks. Maybe there is a better way?

Comment (czpython) [inline djangocms_googlemap/models.py]: yes why not do:

```python
if self.lat and self.lng:
    display += ', {0} / {1} '.format(self.lat, self.lng)
```

Comment (FinalAngel) [inline djangocms_googlemap/models.py]: @czpython then you'd have the problem with the first ,:
image

would have solved it that way if its so easy :D

**Pull request merged**